lib/noticed.rb

```diff
@@ -16,6 +16,7 @@ module DeliveryMethods
     autoload :Database, "noticed/delivery_methods/database"
     autoload :Email, "noticed/delivery_methods/email"
     autoload :Slack, "noticed/delivery_methods/slack"
+    autoload :MicrosoftTeam, "noticed/delivery_methods/microsoft_team"
```

Comment:
Sorry to be nitpicky. Can we pluralize this since Microsoft calls it "Teams" plural?